requirements.txt

```diff
@@ -73,6 +73,7 @@ pyflakes==2.1.1
 PyHamcrest==1.9.0
 pylast==3.1.0
 PyMySQL==0.9.3
+mysqlclient==1.4.2.post1
```

Comment:
Why do we need yet another mysql client?

Reply:
It may just be a fluke, but while testing i kept getting errors when communicating with MariaDB and using the `mysql+pymysql`. i tried and google and people were just using the `mysql`, but in order for me to use that i had to install this client.